packages/react/src/components/tag/Tag.stories.tsx

```diff
@@ -10,109 +11,226 @@ export default {
     controls: { expanded: true },
   },
   args: {
-    children: 'Americum',
+    children: 'Label',
   },
 };
 
-export const DefaultTag = (args) => <Tag {...args} />;
+export const InformativeTagsSmall = (args) => (
+  <div style={{ display: 'flex', gap: '12px', flexWrap: 'wrap', alignItems: 'flex-start' }}>
```

Comment:
Should we also have a TagContainer -component? Tags are about everytime displayed the same way with same gaps etc, so there is a need for a common container.